sources/packages/web/src/types/formio/custom-components/components/number-currency.ts

```diff
@@ -0,0 +1,27 @@
+export const CURRENCY = {
```

Comment:
Please add the new component to the index. 
![image](https://github.com/user-attachments/assets/42e9bb1e-9d7a-4fe5-8515-6dedbf4cc218)

Reply:
added to index!